Dockerfile

```diff
@@ -4,9 +4,9 @@ LABEL maintainer="Chef Software, Inc. <docker@chef.io>"
 ARG EXPEDITOR_CHANNEL
 ARG CHANNEL=stable
 ARG EXPEDITOR_VERSION
-ARG VERSION=4.3.13
+ARG VERSION=3.10.1
 ENV DEBIAN_FRONTEND=noninteractive \
-    PATH=/opt/chefdk/bin:/opt/chefdk/embedded/bin:/root/.chefdk/gem/ruby/2.6.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+    PATH=/opt/chefdk/bin:/opt/chefdk/embedded/bin:/root/.chefdk/gem/ruby/2.5.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
Why the downgrade to the ruby path?

Reply:
ChefDK 3 is still using Ruby 2.5, whereas ChefDK 4 is using Ruby 2.6. I'm guessing the `2.6` got brought in when bringing in some bug fixes from DK 4